<testsuites id="" name="" tests="3" failures="1" skipped="0" errors="0" time="53.721254">
<testsuite name="com.lutron.spec.ts" timestamp="2025-04-29T02:19:43.372Z" hostname="webkit" tests="3" failures="1" skipped="0" time="49.119" errors="0">
<testcase name="verify support link" classname="com.lutron.spec.ts" time="21.059">
<system-out>
<![CDATA[
[[ATTACHMENT|test-results\com.lutron-verify-support-link-webkit\test-finished-2.png]]

[[ATTACHMENT|test-results\com.lutron-verify-support-link-webkit\test-finished-3.png]]

[[ATTACHMENT|test-results\com.lutron-verify-support-link-webkit\test-finished-1.png]]
]]>
</system-out>
</testcase>
<testcase name="verify commercial link" classname="com.lutron.spec.ts" time="11.237">
<system-out>
<![CDATA[
[[ATTACHMENT|test-results\com.lutron-verify-commercial-link-webkit\test-finished-1.png]]
]]>
</system-out>
</testcase>
<testcase name="Test about us link" classname="com.lutron.spec.ts" time="16.823">
<failure message="com.lutron.spec.ts:57:5 Test about us link" type="FAILURE">
<![CDATA[  [webkit] › com.lutron.spec.ts:57:5 › Test about us link @Playwright_With_Jenkins_Practice ────────

    Error: Timed out 5000ms waiting for expect(locator).toHaveText(expected)

    Locator: getByRole('heading', { name: 'Who We Are' })
    Expected string: "Who We Are***"
    Received string: "Who We Are"
    Call log:
      - expect.toHaveText with timeout 5000ms
      - waiting for getByRole('heading', { name: 'Who We Are' })
        5 × locator resolved to <h1 class="hero__tagline">Who We Are</h1>
          - unexpected value "Who We Are"


      61 |     await aboutUs.click();
      62 |     await page.getByLabel('Who We Are').click();
    > 63 |     await expect(page.getByRole('heading', { name: 'Who We Are' })).toHaveText('Who We Are***');
         |                                                                     ^
      64 |
      65 | })
      66 |
        at C:\Users\herra\Desktop\Playwright_Lutron.com\tests\com.lutron.spec.ts:63:69

    Error Context: ..\test-results\com.lutron-Test-about-us-link-webkit\error-context.md
]]>
</failure>
<system-out>
<![CDATA[
[[ATTACHMENT|test-results\com.lutron-Test-about-us-link-webkit\error-context.md]]
]]>
</system-out>
</testcase>
</testsuite>
</testsuites>